src/NosCore.WorldServer/Startup.cs

```diff
@@ -403,7 +412,8 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
             //PacketFactory.Initialize<NoS0575Packet>();
             var optionsBuilder = new DbContextOptionsBuilder<NosCoreContext>()
                 .UseNpgsql(_worldConfiguration.Database!.ConnectionString);
-            DataAccessHelper.Instance.Initialize(optionsBuilder.Options);
+            _dataAccess = new DataAccessHelper();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make the enclosing instance method 'static' or remove this set on the 'static' field.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5326532)